docs/Middleware-Guide.md

```diff
@@ -0,0 +1,82 @@
+# Introduction
+In designing EigenLayer, we aspired to make minimal assumptions about the structure of middlewares built on top of it. If you are getting started looking at the EigenLayer codebase, the Slasher contract contains most of the logic that actually mediates the interactions between EigenLayer and middlewares. Additionally, there is a general-purpose /middleware/ folder, which contains code that can be extended, used directly, or consulted as a reference in building middleware on top of EigenLayer.
+
+## Important Terminology
+### Tasks
+A task in EigenLayer is the discretized unit of work that operators commit to doing when serving a middleware.  
```

Comment:
add formatting to bold the key terms / things we're defining. I think it really helps for readability.

Reply:
Done